codegen/src/main/scala/akka/grpc/gen/Constants.scala

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.gen
+
+object Constants {
+  val DoNotEditComment = "// Generated by Akka gRPC. DO NOT EDIT."
```

Comment:
:+1: 

As the next step could we add the name of the proto file this was generated from (to edit instead)?